### DISCUSSION TITLE
403 Issue while connecting to copilot in VSCode

### DISCUSSION THREAD

**Discussion by dht** [community/community#51605] [Copilot Conversations]
### Select Topic Area

Bug

### Body

```
[INFO] [auth] [2023-03-31T19:04:42.105Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2023-03-31T19:04:42.107Z] GitHub Copilot could not connect to server. Extension activation failed: "At this time, Copilot is not available in your location"
```

It worked in the last 6 months. All these did not fix the issue:
1. Restarting VSCode
2. Logging out & in of extension 
3. Logging out & in of  Github in VSCode 
4. Installing and uninstalling extension

**Comment by mcglabs** [comment]
I'm having the same issue. Worked the past few months, then today this issue. In the US so not sure what could be the issue, maybe a system issue. Hope we can get an answer and it's resolved soon.

**Comment by unspectd** [comment]
It just stopped working 2 days ago with the same logs, but in Goland. So it seems like the problem occurs across different plugins and may be related to infra.
Unfortunately there is no support from GitHub related to these hiccups.

**Comment by Akash1134** [comment]
Hi @dht and @mcglabs ,

Thanks for getting in touch.

Due to current trade controls, your GitHub account has been restricted from making use of Copilot. This is a region-based restriction and applies to all users located in some trade-restricted countries, being flagged for the same earlier, and is separate from other GitHub services. You can continue to use other GitHub services including your Pro plan. You can read more about it here:

[GitHub and Trade Controls](https://docs.github.com/en/site-policy/other-site-policies/github-and-trade-controls)

Sorry if that wasn't what you were hoping to hear. 

If you feel this should not be the case, feel free to contact our Support team and [open a ticket ](https://support.github.com/request?tags=dotcom-footer)with them. Thanks!

**Comment by Jason-K** [comment]
I'm also experiencing this issue. In US and with an active Copilot subscription.
